src/components/Section/SectionView.scss

```diff
@@ -1,22 +1,18 @@
 @import 'eqip-colors';
 
 .section-view {
-  max-width: 70rem;
+  max-width: 80rem;
```

Comment:
Since we are using 80rem as the max width for the content area and saved-indicator, we could probably turn that into a variable as well.

Reply:
Yeah I'm thinking that would be a larger refactoring effort. From poking around in the styles for a while I get the feel that there is _**a lot**_ we can refactor.